### PR TITLE
Add Random UUID expression

### PR DESCRIPTION
**Pull request by Mr-Darth**
### Description

Added an expression to get a random UUID.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprRandomUUID.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+@Examples({"set {_uuid} to random uuid"})
+@Since("INSERT VERSION")
+public class ExprRandomUUID extends SimpleExpression<String> {
+	
+	static {
+		Skript.registerExpression(ExprRandomUUID.class, String.class, ExpressionType.SIMPLE, "[a] random uuid");
+	}
+	
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, SkriptParser.ParseResult parseResult) {
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/expressions/ExprRandomUUID.java:49]: I know this sounds nitpicky, but you could please import ParseResult directly.

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
Just a small change request

File: src/main/java/ch/njol/skript/expressions/ExprRandomUUID.java (hunk 1 above)
Comment (APickledWalrus) [reply src/main/java/ch/njol/skript/expressions/ExprRandomUUID.java:49]: They don't have to override init at all since they're just returning true.

Comment (ShaneBeee) [reply src/main/java/ch/njol/skript/expressions/ExprRandomUUID.java:49]: ah yeah, thats true as well.

**Comment by APickledWalrus** [comment]
I think you can just remove `init` entirely since you're not doing anything with it.

**Comment by Mr-Darth** [comment]
> I think you can just remove `init` entirely since you're not doing anything with it.

If I remove it, I get this error:
```
The type ExprRandomUUID must implement the inherited abstract method SyntaxElement.init(Expression<?>[], int, Kleenean, SkriptParser.ParseResult)
```

**Comment by APickledWalrus** [comment]
> 
> 
> > I think you can just remove `init` entirely since you're not doing anything with it.
> 
> If I remove it, I get this error:
> 
> ```
> The type ExprRandomUUID must implement the inherited abstract method SyntaxElement.init(Expression<?>[], int, Kleenean, SkriptParser.ParseResult)
> ```

This should be good to go then.

**Review by APickledWalrus — DISMISSED** [review]

**Review by Wealthyturtle — APPROVED** [review]

**Review by Pikachu920 — DISMISSED** [review]

**Review by ShaneBeee — APPROVED** [review]

**Comment by ShaneBeee** [comment]
Thank you for your contribution 👍

**Pull request merged**